.env.example

```
@@ -26,3 +26,8 @@ RESEND_FROM=noreply@yourdomain.resend.dev
 
 # Resend API key is required
 RESEND_API_KEY=your_resend_api_key
+
+GMAIL_EMAIL=your_gmail_email@gmail.com
+
+# Your gmail app password
+GMAIL_APP_PASSWORD=xxxx xxxx xxxx xxxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)